app/controllers/target_dashboard_controller.rb

```diff
@@ -25,6 +25,34 @@ def index
 
     @country_and_regions_headings =
       Aichi11TargetDashboardSerializer.new.serialize_head
+
+    @config_carousel_global = {
```

Comment:
I'm not sure this is the best location for this, I would move it into a helper maybe??